test/acceptance/interfaces/bdd.spec.js

```diff
@@ -32,3 +32,33 @@ context('test suite', function () {
     expect(this.number).to.equal(5);
   });
 });
+
+describe('a suite skipped by "before" hook', function () {
+  before(function () {
+    this.skip();
+  });
+
+  it('should skip this test', function () {
+    expect(2).to.equal(1);
```

Comment:
In all the failure cases, you should call fail directly: `expect().to.fail('message here')`